src/test/extension.test.ts

```diff
@@ -565,8 +611,6 @@ suite('Extension Basic Tests', () => {
         let config = getPeacockWorkspaceConfig();
         assert.ok(!config[ColorSettings.activityBar_badgeBackground]);
         assert.ok(!config[ColorSettings.activityBar_badgeBackground]);
-
-        await updateAffectedElements(allAffectedElements);
```

Comment:
This probably still should be here to reset the state before leaving the test

Reply:
thanks. that worked.

probably should be in setup or teardown moving forward

Reply:
So to do specific setup for a single test should it be wrapped in a `suite()` where `suiteSetup()` and `suiteTeardown()` are available?

Reply:
Thinking out loud:

1. suite would mean doing it once before and after the suite

2. test level is before and after each test

3. if it only affect one test, inside the test is ok.

I occasionally review the tests to see where these are. I think a refactor of the rest may b in order soon. There are a ton of them in one massive suite. I may want to create a few suites that call reusable functions to do setup/teardown